.gitignore

```diff
@@ -127,3 +127,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+xnat_admin_tools/get_result.py
```

Comment:
This is supposed to be run locally and not needed in the container. I will add it to the repo though, so other people can access the result. This would be useless once we have the rq-dashboard working though.

Reply:
I think it can come in handy. Also the admin tools can include more things that are not directly running in the relay that can help us with the admin tasks